calliope/config/defaults.yaml

```diff
@@ -46,7 +46,8 @@ model:
     timeseries_data: null  # Dict of dataframes with time series data (when passing in dicts rather than YAML files to Model constructor)
     timeseries_dateformat: '%Y-%m-%d %H:%M:%S'  # Timestamp format of all time series data when read from file
     file_allowed: ['clustering_func', 'energy_eff', 'energy_ramping', 'export', 'om_con', 'om_prod', 'parasitic_eff', 'resource', 'resource_eff', 'storage_loss', 'carrier_ratios']  # List of configuration options allowed to specify "file=" to load timeseries data. This can be updated if you're adding a new custom constraint that requires a newly defined parameter to be a timeseries. If updating existing parameters, you can expect existing constraints to not change behaviour or to break on being constructed.
-
+    optimisation_config_overrides: []  # List of custom-constraints
```

Comment:
```suggestion
    optimisation_config_overrides: []  # List of references to optimisation problem overrides. References can be made to in-built overrides (without a `.yaml` suffix) and to user-defined override YAML files (at paths relative to the loaded `model.yaml` file)
```

Reply:
Call this something other than `overrides` since overrides already exist elsewhere? Though in a sense this is the same because underneath it's also munging together dicts. Still - something like `custom_config`? `optimisation` seems somewhat superfluous too since we're in an optimisation modelling framework anyway?

Reply:
agreed offline: `custom_math`